src/schema/v2/conversation/index.ts

```diff
@@ -322,7 +322,10 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
           })
 
           return {
-            collectorProfile: data.collector_profile,
+            collectorProfile: {
+              ...data.collector_profile,
+              partnerId: to_id,
```

Comment:
I agree that there is a long way from the conversation to reach the interestConnection, and I also agree that this is necessary to make it work.

Reply:
maybe should we comment this section to make it clear what we use it for?